zerver/lib/presence.py

```diff
@@ -99,6 +99,11 @@ def get_legacy_user_presence_info(
         timestamp=most_recent_info["timestamp"],
     )
 
+    # Include a copy of the aggregated field under the "default" key for backward
+    # compatibility with the mobile application, which expects to find data in this
+    # format under some client name.
+    result["default"] = result["aggregated"].copy()
+
     result["website"] = most_recent_info
```

Comment:
Also just wanted to highlight here that we are putting new-format presence info data under the `website` key, which is kind of wonky - in case we want to make any further changes here

Reply:
I think not clear to me why what we put in `result["website"] ` is not sufficient here. Should we perhaps doing just that and calling that `default` instead of `website`?

Reply:
Yeah, it sounds like we had already thought of this detail and just picked a different name. Let me test-deploy this without this final commit, and we can discuss in "api design".